core/views/users.py

```diff
@@ -0,0 +1,29 @@
+from django.shortcuts import render
+
+URL = '../templates/user'
+
+def home(request):
+    return render(request, '../templates/user/index.html')
+
+def profile(request, id):
+    users = [
+        {"id": 1, "name": "John Doe"},
+        {"id": 2, "name": "Helgi Hak"},
+    ]
+    ctx = None
+    for user in users:
```

Comment:
Myndi abstract-a þetta í fall sem finnur sækir userID en þarft svo sem að breyta þessu þegar við byrjum að sækja í model-in